MigrationV3V4/Node/Skills/v4-root-bot/.env.example

```diff
@@ -2,10 +2,10 @@ MicrosoftAppId=
 MicrosoftAppPassword=
 SkillHostEndpoint=
```

Comment:
Since the readme explains how to use the emulator, and we have pre-defined local paths ... i think it makes sense to add the endpoints to this .env

Reply:
agreed. good call. done